<launch>
    <arg name="empty" default="true"/>
    <node if="$(arg empty)" pkg="map_server" name="map_server" type="map_server" args="$(find bluerov_control)/launch/params/emptymap.yaml" output="screen"/>

    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

        <rosparam file="$(find bluerov_control)/launch/params/move_base/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find bluerov_control)/launch/params/move_base/costmap_common_params.yaml" command="load" ns="local_costmap" />

        <rosparam file="$(find bluerov_control)/launch/params/move_base/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find bluerov_control)/launch/params/move_base/global_costmap_params.yaml" command="load" />

        <rosparam file="$(find bluerov_control)/launch/params/move_base/dwa_local_planner_params.yaml" command="load" />
        <rosparam file="$(find bluerov_control)/launch/params/move_base/move_base_params.yaml" command="load" />

        <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
        <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
    </node>

</launch>